config/non-admin-controller_rbac/role_binding.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    app.kubernetes.io/name: clusterrolebinding
+    app.kubernetes.io/instance: non-admin-manager-rolebinding
+    app.kubernetes.io/component: rbac
+    app.kubernetes.io/created-by: oadp-operator
+    app.kubernetes.io/part-of: oadp-operator
+    app.kubernetes.io/managed-by: kustomize
+  name: non-admin-manager-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: non-admin-manager-role
+subjects:
+- kind: ServiceAccount
+  name: non-admin-controller-manager
```

Comment:
I believe this should be OADP Operator SA and all the non-admin related CRD perms would be added to operator SA ? in CSV ? Let me know if I am thinking down an incorrect path.